functions/payment.js

```diff
@@ -1,282 +0,0 @@
-const admin = require("firebase-admin");
-const functions = require("firebase-functions");
-
-const PROJECT_ID = functions.config().project.id;
-const STRIPE_CONFIG = functions.config().stripe;
-
-// Set your secret key. Remember to switch to your live secret key in production!
-// See your keys here: https://dashboard.stripe.com/account/apikeys
-const stripe = require("stripe")(STRIPE_CONFIG.secret_key);
-
-const PURCHASE_TABLE = "purchases";
-const CUSTOMER_TABLE = "customers";
```

Comment:
Are these collections still referred to anywhere in the platform code/firebase rules/etc at all?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40